cmd/neofs-adm/internal/modules/morph/initialize_deploy.go

```diff
@@ -372,11 +372,17 @@ func (c *initializeContext) readContracts(names []string) error {
 func readContract(ctrPath, ctrName string) (*contractState, error) {
 	rawNef, err := os.ReadFile(filepath.Join(ctrPath, ctrName+"_contract.nef"))
 	if err != nil {
-		return nil, fmt.Errorf("can't read NEF file for %s contract: %w", ctrName, err)
+		rawNef, err = os.ReadFile(filepath.Join(ctrPath, "contract.nef"))
```

Comment:
go this way only if file not found? also, do you play to drop it in the future?

Reply:
@carpawell is faster, but i'll leave my comment anyway

err may be other than 404, so i'd either:

1. try other file on 404 only
2. try anyway, but keep both errors when it fails too

same for manifest files

Reply:
> also, do you plan to drop it in the future?

why do we try to support both ways? how long?

Reply:
Because it still is a generic tool. We'll see. An issue can be created.